Makefile

```diff
@@ -1,52 +1,22 @@
-all: ci rhel
+ci_keys = keys/verifier-public-key-openshift-ci keys/verifier-public-key-openshift-ci-2
+ci_stores = stores/store-openshift-ci-release
+ci_manifests = manifests/0000_90_cluster-update-keys_configmap.yaml
+rhel_keys = keys/verifier-public-key-redhat-release keys/verifier-public-key-redhat-beta-2
+rhel_stores = stores/store-openshift-official-release stores/store-openshift-official-release-mirror
+rhel_manifests = manifests.rhel/0000_90_cluster-update-keys_configmap.yaml
+
+all: hack/generate-configmap.sh ci rhel
 .PHONY: all
 
 # The CI system uses the OpenShift CI public key and verifies it against a bucket on GCS.
-ci:
-	keydir=$(shell mktemp -d -t keys-XXXXXXXX); \
-	gpg --dearmor < keys/verifier-public-key-openshift-ci > "$$keydir/verifier-public-key-ci.gpg"; \
-	gpg --dearmor < keys/verifier-public-key-openshift-ci-2 >> "$$keydir/verifier-public-key-ci.gpg"; \
-	gpg --enarmor < "$$keydir/verifier-public-key-ci.gpg" > "$$keydir/verifier-public-key-ci"; \
-	sed -i 's/ARMORED FILE/PUBLIC KEY BLOCK/' "$$keydir/verifier-public-key-ci"; \
-	echo "# Release verification against OpenShift CI keys signed by the CI infrastructure" > \
-		manifests/0000_90_cluster-update-keys_configmap.yaml; \
-	oc create configmap release-verification \
-			--from-file=$$keydir/verifier-public-key-ci \
-			--from-file=stores/store-openshift-ci-release \
-			--dry-run -o yaml | \
-		oc annotate -f - release.openshift.io/verification-config-map= \
-			include.release.openshift.io/ibm-cloud-managed="true" \
-			include.release.openshift.io/self-managed-high-availability="true" \
-			include.release.openshift.io/single-node-developer="true" \
-			-n openshift-config-managed --local --dry-run=client -o yaml	>> \
-		manifests/0000_90_cluster-update-keys_configmap.yaml; \
-	echo "  namespace: openshift-config-managed" >> \
-		manifests/0000_90_cluster-update-keys_configmap.yaml
+ci: $(ci_keys) $(ci_stores) $(ci_manifests)
```

Comment:
this rule generates the CI manifest, it doesn't consume it.  As it stands:

```console
$ git --no-pager log --oneline -1
d8197c0 (HEAD, origin/pr/39) Add script generating configmaps
$ rm -f manifests*/*yaml
removed 'manifests.rhel/0000_90_cluster-update-keys_configmap.yaml'
removed 'manifests/0000_90_cluster-update-keys_configmap.yaml'
$ make
make: *** No rule to make target 'manifests/0000_90_cluster-update-keys_configmap.yaml', needed by 'ci'.  Stop.
```

There are at least two possible solutions:

* Drop `$(ci_manifests)` here (and similarly for the `rhel` target).  Because of the `.PHONY` entries, Make will try and rebuild the manifests each run, but rebuilding them is not all that costly, so not a big deal.
* Create a rule that uses the target manifest filename as the target, and have the `.PHONY` alias depend on that.  So Make knows when it has to do the rebuild, and when a rebuild would just be cycles creating identical content.  Something like:
    ```make
    ci: $(ci_manifests)
    .PHONY: ci
    $(ci_manifests): hack/generate-configmap.sh $(ci_keys) $(ci_stores)
      $< --ci
    ```